Formula/snappy.rb

```diff
@@ -1,8 +1,8 @@
 class Snappy < Formula
   desc "Compression/decompression library aiming for high speed"
   homepage "https://google.github.io/snappy/"
-  url "https://github.com/google/snappy/archive/1.1.8.tar.gz"
-  sha256 "16b677f07832a612b0836178db7f374e414f94657c138e6993cbfc5dcc58651f"
+  url "https://github.com/google/snappy/archive/1.1.9.tar.gz"
+  sha256 "75c1fbb3d618dd3a0483bff0e26d0a92b495bbe5059c8b4f1c962b478b6e06e7"
```

Comment:
```suggestion
  url "https://github.com/google/snappy.git",
      tag:      "1.1.9",
      revision: "2b63814b15a2aaae54b7943f0cd935892fae628f"
```
You probably want to try this.

Reply:
Oh yeah! Sure! But I was a little reluctant to change the download strategy entirely (I’m wondering why now 🤦‍♂️)
Sorry about the confusion!

Reply:
> Running over there rn to give them a heads up about that holey archive…

This is to be expected when dealing with submodules, not really an issue with the archives.

Reply:
Thanks for the heads-up 😁 
Newbies like me need as much as we can get 😆